### ISSUE TITLE
Update the colour for the shepherding screen

### ISSUE THREAD

**Issue by samjmartin04** [PeterJCLaw/srcomp-screens#14]
Currently the colours on the shepherding screen are quite hard to distinguish so it would be good to update these to be clearer.

**Comment by PeterJCLaw** [comment]
@samjmartin04 has this been any better for SR2025? The colours are determined by the compstate (not this software), so it's easy to change.

**Comment by samjmartin04** [comment]
Yes, much better. The people I've spoken to have found that more distinct colours (plus them matching with the map provided in the checklists) has made shepherding teams much easier.
